bucket/ffmpeg-shared.json

```
@@ -0,0 +1,35 @@
+{
+    "version": "4.1.3",
+    "homepage": "https://ffmpeg.zeranoe.com/builds/",
+    "license": "GPL-3.0",
```

Wrong identifier

---

```suggestion
    "license": {
        "identifier": "GPL-3.0-only"
    },
```
Is it better this way?

---

Already fixed with https://github.com/scoopinstaller/scoop-main/commit/ab92aa791c959d26be824986844d571c2fc45286 :smile:

---

No. Do not use object unless you put URL.

Just use `"license": "GPL-3.0-only"`